dask_jobqueue/core.py

```diff
@@ -161,7 +161,7 @@ def start_workers(self, n=1):
         for _ in range(n):
             with self.job_file() as fn:
                 out = self._call(shlex.split(self.submit_command) + [fn])
-                job = out.decode().split('.')[0].strip()
+                job = self._jobid_from_submit_output(out)
```

Comment:
I think you should have `_job_id_from_submit_output` defined in `JobQueueCluster`. It should raise a `NotImplementedError` saying that if you inherit from `JobQueueCluster` you need to implement `_job_id_from_submit_output` that converts the submit_command stdout output to the job id (or a improved version of this).